bin/liquidator/src/rebalance.rs

```diff
@@ -324,7 +334,11 @@ impl Rebalancer {
             // to sell them. Instead they will be withdrawn at the end.
             // Purchases will aim to purchase slightly more than is needed, such that we can
             // again withdraw the dust at the end.
-            let dust_threshold = I80F48::from(2) / token_price;
+            let dust_threshold = if self.config.use_limit_order {
+                I80F48::from(self.dust_threshold_for_limit_order(token).await?)
```

Comment:
it's possible for this (and the unwind call later) to fail consistently if there is no available market for the token. I think a failure here should not prevent rebalancing of other tokens